CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changes
 4.4.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for general-purpose Python 3.x annotations / type-hints in interface definitions.
```

Comment:
I would also prefer a less ambitious changelog entry, similar to the one which added `__qualname__` to the list of skipped attrs in version 4.0.0.  For reference, that messsage was:

```rst
Avoid exceptions due to tne new ``__qualname__`` attribute added in Python 3.3 (see PEP 3155 for rationale). 
```